docs/api/combining_optimizers.rst

```diff
@@ -15,4 +15,3 @@ Multi-transform
 ~~~~~~~~~~~~~~~
 .. autofunction:: multi_transform
 .. autoclass::  MultiTransformState
```

Comment:
Add a one line docstring to MultiTransformState and add MultiTransformState to the autosummary